src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -590,39 +724,68 @@ const Packages = () => {
     packages,
     toggleSelected,
     toggleSourceRepos,
-  ]).sort((a, b) => {
-    const aPkg = a.name.toLowerCase();
-    const bPkg = b.name.toLowerCase();
-    // check exact match first
-    if (aPkg === debouncedSearchTerm) {
-      return -1;
-    }
-    if (bPkg === debouncedSearchTerm) {
-      return 1;
-    }
-    // check for packages that start with the search term
-    if (
-      aPkg.startsWith(debouncedSearchTerm) &&
-      !bPkg.startsWith(debouncedSearchTerm)
-    ) {
-      return -1;
+  ]).sort((a, b) => sortfn(a.name, b.name));
+
+  const transformedGroups = useMemo(() => {
```

Comment:
Something probably doesn't get "cleaned" properly. When searching for a package and a group the results will start to mix

![mixed-results](https://github.com/osbuild/image-builder-frontend/assets/49452678/dcbaa509-a8a9-412a-9d49-3ac8bc37b808)